README.md

```diff
@@ -1,10 +1,42 @@
 # Temporal documentation
 
+Temporal is constantly working to improve and expand its documentation.
+Some components may change without notice.
+Page slugs (URLs), menu labels, and the location of information are a few of the items you can expect to see altered as we endeavor to give you the best experience possible.
+
 The documentation in this repository is published to https://docs.temporal.io.
 
-The source of this documentation is available to the public for individual and commercial use.
+If information you are looking for seems to be missing, visit the [Temporal community](https://community.temporal.io/) forum for help.
 
-Please read the [CONTRIBUTING](/CONTRIBUTING.md) guide before you submit any pull requests.
+Please read the [CONTRIBUTING](/CONTRIBUTING.md) guide and the [STYLE](/STYLE.nd) guide before you submit any pull requests.
 
 Maintainers and contributors to this project are expected to conduct themselves in a respectful way.
 See the [CNCF Community Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) as a reference.
+
+The source of this documentation is available to the public for individual and commercial use.
+
+MIT License
```

Comment:
We might want to put this as a stand alone file and then link to it.

Reply:
Done. I added LICENSE.md to this PR.